docs/kop.md

```diff
@@ -0,0 +1,139 @@
+---
+dockerfile: "https://hub.docker.com/r/streamnative/kop(TBD)"
+---
+
+KoP (Kafka on Pulsar) brings the native Apache Kafka protocol support to Apache Pulsar by introducing a Kafka protocol handler on Pulsar brokers. By adding the KoP protocol handler to your existing Pulsar cluster, you can migrate your existing Kafka applications and services to Pulsar without modifying the code. This enables Kafka applications to leverage Pulsar’s powerful features, such as:
+
+- Streamlined operations with enterprise-grade multi-tenancy
+- Simplified operations with a rebalance-free architecture
+- Infinite event stream retention with Apache BookKeeper and tiered storage
+- Serverless event processing with Pulsar Functions
+
+KoP, implemented as a Pulsar [protocol handler](https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/protocol/ProtocolHandler.java) plugin with the protocol name "kafka", is loaded when Pulsar broker starts. It helps reduce the barriers for people adopting Pulsar to achieve their business success by providing a native Kafka protocol support on Apache Pulsar. By integrating the two popular event streaming ecosystems, KoP unlocks new use cases. You can leverage advantages from each ecosystem and build a truly unified event streaming platform with Apache Pulsar to accelerate the development of real-time applications and services.
+
+KoP implements the Kafka wire protocol on Pulsar by leveraging the existing components (such as topic discovery, the distributed log library - ManagedLedger, cursors and so on) that Pulsar already has.
+
+The following figure illustrates how the Kafka-on-Pulsar protocol handler is implemented within Pulsar.
+
+![](kop-architecture.png)
```

Comment:
Using github's image bed is a better and simpler option

Reply:
Do you mean use this link directly? https://github.com/streamnative/kop/blob/84bb7e8222867e3f298b9a552fec939c3ab1d2e0/docs/kop-architecture.png
If yes, shall we adopt the same way for other links in the kop.md file?

Reply:
@tuteng I've added a yaml file based on the [workflow](https://github.com/streamnative/pulsar-io-template/blob/master/README.md#doc). After I add a test image in the kop.md file, shall we merge this PR and check the sync effect on Hub?

Reply:
Is there any difference between syncing connector and syncing kop?